### PR TITLE
API-7225/node-versioning

### PR DESCRIPTION
**Pull request by ediiotero**
Story: [API-7225](https://vajira.max.gov/browse/API-7225)

Adding version files for both nvm and asdf, also updated readme

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,14566 +1,8 @@
 {
   "name": "loast",
   "version": "0.0.0-development",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
Comment (alexandervalencia) [inline package-lock.json:4]: `package-lock.json` should stay version 2

**Review by alexandervalencia — CHANGES_REQUESTED** [review]
Once the package-lock.json changes are reverted this will be good to go!

File: package-lock.json (hunk 1 above)
Comment (ediiotero) [reply package-lock.json]: I totally misread that. Thanks

**Review by alexandervalencia — APPROVED** [review]
Thanks @ediiotero ! 🚀

**Pull request merged**